crates/typed-store/src/metrics.rs

```diff
@@ -172,3 +327,22 @@ impl DBMetrics {
         }
     }
 }
+
+#[derive(Debug)]
+pub struct DBMetrics {
+    pub op_metrics: OperationMetrics,
+    pub cf_metrics: ColumnFamilyMetrics,
+}
+
+impl DBMetrics {
+    pub(crate) fn new(registry: &Registry) -> Self {
+        DBMetrics {
+            op_metrics: OperationMetrics::new(registry),
+            cf_metrics: ColumnFamilyMetrics::new(registry),
+        }
+    }
+    pub fn make_db_metrics(registry: &Registry) -> &'static Arc<DBMetrics> {
+        static ONCE: OnceCell<Arc<DBMetrics>> = OnceCell::new();
+        ONCE.get_or_init(|| Arc::new(DBMetrics::new(registry)))
+    }
```

Comment:
I suppose this could be fixed by creating separate metrics for CF's instead of using CF as a dimension to the metric. But placing a comment to address should be fine for now.